cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -77,6 +77,26 @@
     {% set show_categories = cat_controls.show_preview_categories if cat_controls is defined else true %}
     <article class="o-post-preview">
         <div class="m-meta-header">
+            {% if show_categories and post.categories.exists()  %}
+            <div class="m-meta-header_left">
+                {# Newsroom Blog category logic #}
```

Comment:
Consider wording it more specifically like the suggestion below, and consider moving it inside the `if` statement two lines below.

```suggestion
                {# If rendering a blog post in the newsroom, use the 'Blog' aggregate category instead of the post's true category. #}
```